pkgs/applications/emulators/mame/default.nix

```diff
@@ -156,7 +157,10 @@ stdenv.mkDerivation rec {
   # to the final package after we figure out how they work
   installPhase =
     let
-      icon = "${papirus-icon-theme}/share/icons/Papirus/32x32/apps/mame.svg";
+      icon = fetchurl {
```

Comment:
Why this change?

Reply:
`papirus-icon-theme` supports only Linux https://github.com/NixOS/nixpkgs/blob/320649722ab1d50ab19540b1c9113fc7f789e6f0/pkgs/data/icons/papirus-icon-theme/default.nix#L65